letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -262,14 +262,6 @@ BootstrapRpmCommon() {
       echo "Could not install additional dependencies. Aborting bootstrap!"
       exit 1
   fi
-
-
-  if $SUDO $tool list installed "httpd" >/dev/null 2>&1; then
```

Comment:
This change needs to be made in the file `letsencrypt-auto-source/pieces/bootstrappers/rpm_common.sh` (running `build.py` in `letsencrypt-auto-source/` will propagate the change).